### ISSUE TITLE
The log-in form isn't intuitive enough

### ISSUE THREAD

**Issue by Changaco** [liberapay/liberapay.com#1535]
[The log-in form](https://liberapay.com/log-in) states what a user should do if they've lost their password, but we regularly receive emails from users asking how to reset their passwords, so it's not intuitive enough.

![Screenshot of the log-in form](https://user-images.githubusercontent.com/1581590/61724364-1d3c2700-ad6e-11e9-99a1-8c7098647ece.png)

**Comment by trebmuh** [comment]
Probably people are used to a "forgot password?" link.

**Comment by Changaco** [comment]
My thoughts exactly. It's possible that rewording and restyling the help text so that it contains the words "lost your password" in bold would be enough to solve the problem. Alternatively we could create a separate password-less form and link to it.

**Comment by trebmuh** [comment]
What about:
> If you **forget or lost your password**, leave the password field empty. We'll send you a login via email.

**Comment by mattbk** [comment]
How are "forget" and "lost" different?

Could it be "If you need to reset your password, leave the password field empty. We'll send you a login via email." ? This removes the guess at why they need a password and just gives them the action they want to perform.

**Comment by trebmuh** [comment]
fair enough

**Comment by Changaco** [comment]
We don't use the word "forget" because people who use password managers don't forget their passwords, although they can still lose them sometimes.

We could only invite the user to "reset" their password, but strictly speaking they can't reset it if they never set it in the first place, so it's slightly weird.

**Comment by sim6** [comment]
What about (including bold):
If you need to **reset or set-up your password**, leave the password field empty. We'll send you a login via email.

**Comment by justin-hubbard** [comment]
If there's some consensus on what the fix should be, I'm looking for an issue to solve. I'm trying to get into open source and I'm looking for simple fixes to submit so I can get used to the process.

**Comment by Changaco** [comment]
@justin-hubbard Feel free to open a pull request, there doesn't need to be consensus on the exact wording before a pull request is opened.

At this point my suggestion is: “If you've {bold}lost your password{end_bold}, or if your account doesn't have a password at all, then leave the password field empty. We'll send you a login link via email.”

**Comment by Changaco** [comment]
As I feared, #1594 wasn't sufficient to fix the problem. We still receive emails from confused people.

**Comment by mattbk** [comment]
Maybe people don't get that they need to enter their email address. Would wording like this help?

> If you've **lost your password**, or if your account doesn't have a password at all, then enter your email address and leave the password field empty. We'll send you a login link via email.

**Comment by Changaco** [comment]
I don't think we can fix this by rewording the help text again, because the problem is that some people simply don't read it.

We probably need a solution that pushes the user click on something, because everyone reads button labels.

**Comment by mattbk** [comment]
Wire up a button that says "click here to reset password" and just hides the existing text and the password box?

**Comment by mattbk** [comment]
Discourse has a link pop up when you start entering a username/email address. Even if you don't want to fool with that, the wording and placement of the link are nice ("Skip the password; email me a login link"):
<img width="667" alt="image" src="https://user-images.githubusercontent.com/3729038/133653532-a6a20f6a-465c-4074-a338-335e59a99d89.png">
